### DISCUSSION TITLE
(solved) Install Fail: Netbox 3.7.2 on Ubuntu 22.04.3

### DISCUSSION THREAD

**Discussion by maxzapp** [netbox-community/netbox#15160] [Q&A]
Hi i run througth the install docu until i should test the installaton with pythons internes server. I got the following message:
----------------
netbox$ python3 manage.py runserver 0.0.0.0:8000 --insecure
Traceback (most recent call last):
  File "/opt/netbox-3.7.2/netbox/manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'
----------------------------

There is no dedicated django installation mentioned.

Thx for any help

**Comment by a084ed22** [comment]
You need to activate the venv

**Reply by maxzapp** [reply]
ahhhrrrggghh i didnt really read the tip section many thanks
